group/mod/int.go

```diff
@@ -362,7 +362,7 @@ func (i *Int) UnmarshalFrom(r io.Reader) (int, error) {
 	return marshalling.ScalarUnmarshalFrom(i, r)
 }
 
-// BigEndian encodes the value of this Int into a big-endian byte-slice
+// kyber.BigEndian encodes the value of this Int into a big-endian byte-slice
```

Comment:
This seems like a mistake during replace

Reply:
Nice catch ! yup its a mistake